shared/chat/inbox/container.js

```diff
@@ -46,26 +46,26 @@ const passesParticipantFilter = (filter: string, participants: Array<string>, yo
 // Simple score for a filter. returns 1 for exact match. 0.75 for full name match
 // in a group conversation. 0.5 for a partial match
 // 0 for no match
-function scoreFilter(filter, stringToFilterOn, you) {
-  if (stringToFilterOn.indexOf(',') > 0) {
-    const participants = stringToFilterOn.split(',')
-    if (participants.some(p => p.toLowerCase() === filter.toLowerCase())) {
+function scoreFilter(filter: string, stringToFilterOn: string, participants: Array<string>, you: string) {
+  const lcFilter = filter.toLowerCase()
+  if (participants.length) {
+    if (participants.some(p => p.toLowerCase() === lcFilter)) {
       if (participants.length === 2) {
```

Comment:
maybe the score should be a function of how many people (less is better) so if you're looking for yourself its a 1, else its like 1 - ((numParticipants / 100) * 0.25) or something